letsencrypt/config.yaml

```diff
@@ -20,7 +20,6 @@ options:
   domains:
     - null
   email: null
-  keytype: ecdsa
```

Comment:
I think for all those add-ons which got started at least once, the default is already in the option store of the add-on.

But not much we can do about it. At least the ones which get started later will start with an empty default, which should lead to the auto-detection to work. For everyone else, they got auto-migrated to `ecdsa`.

Reply:
I'll try testing that in a VM, but yeah we can't go back in time to change that.

Reply:
Yeah it's just stuck in there now if the config was saved while it was part of the default config.

We could go back to `key_type` and delete `keytype`. Because of auto-detect it wouldn't break anything if anyone had changed it to want RSA.

Reply:
Yeah I like the way it works clearing `keytype`. Nothing breaks and auto-detect will work for everyone.